src/Symfony/Component/Console/Style/OutputStyle.php

```diff
@@ -78,6 +78,12 @@ public function getFormatter(): OutputFormatterInterface
         return $this->output->getFormatter();
     }
 
+    public function isSilent(): bool
+    {
+        // @deprecated since Symfony 7.2, change to $this->output->isSilent() in 8.0
+        return method_exists($this->output, 'isSilent') ? $this->output->isSilent() : false;
```

Comment:
this should rather use `$this->output->getVerbosity() === self::VERBOSITY_SILENT` as fallback.